modules/i3s/package.json

```diff
@@ -35,6 +35,7 @@
     "@loaders.gl/draco": "3.0.0-alpha.12",
     "@loaders.gl/images": "3.0.0-alpha.12",
     "@loaders.gl/loader-utils": "3.0.0-alpha.12",
+    "@loaders.gl/polyfills": "^2.3.13",
```

Comment:
Yes. I already included `@loaders.gl/polyfills` to the I3S module just because `Promise.allSettled()` method is supported pretty much by newest versions of browsers. These changes are only for major browsers and not for IE just because IE doesn't support `Promise.all()` method  as well and we don't create a polyfill for that functionality.
So there are two possible options:

1. Don't create a polyfill for `Promise.allSettled()` at all just because we support only recent versions of the major evergreen browsers

![image](https://user-images.githubusercontent.com/70207219/116034629-450b9d80-a66c-11eb-8241-57addb7fa697.png)

2. Land this PR which allows to support this method in I3S mudule.